src/main/resources/assets/skyblocker/lang/en_us.json

```diff
@@ -214,8 +214,10 @@
   "text.autoconfig.skyblocker.option.locations.dungeons.croesusHelper": "Croesus Helper",
   "text.autoconfig.skyblocker.option.locations.dungeons.croesusHelper.@Tooltip": "Gray out chests that have already been opened.",
   "text.autoconfig.skyblocker.option.locations.dungeons.enableMap": "Enable Map",
-  "text.autoconfig.skyblocker.option.locations.dungeons.mapScreen": "Dungeon Map Placement Config...",
+  "text.autoconfig.skyblocker.option.locations.dungeons.enableScore": "Enable Score HUD",
+  "text.autoconfig.skyblocker.option.locations.dungeons.mapScreen": "Dungeon Map & Score Placement Config...",
   "text.autoconfig.skyblocker.option.locations.dungeons.mapScaling": "Map Scaling",
+  "text.autoconfig.skyblocker.option.locations.dungeons.scoreScaling": "Score Scaling",
```

Comment:
Translation key should match the path to the config, i.e. "text.autoconfig.skyblocker.option.locations.dungeons.dungeonScore.enableScoreHud"